plugin/types.go

```diff
@@ -125,8 +120,43 @@ type Attributes struct {
 // SizeUnknown can be used to denote that the size is unknown and should be queried from content.
 const SizeUnknown = ^uint64(0)
 
-// The Registry contains the core filesystem data.
-// Plugins: maps plugin mount points to their implementations.
+// Registry represents the plugin registry. It is also Wash's root.
```

Comment:
I'd put this n a separate file now. It's something plugin writers don't need to know, and I've been trying to keep types.go constrained to things you use when writing plugins.